### ISSUE TITLE
alpine 3.14.2 reported as unsupported

### ISSUE THREAD

**Issue by AmaliMatharaarachchi** [aquasecurity/trivy#1220]
## Description

alpine 3.14.2 is the latest supported version of alpine linux.
https://alpinelinux.org/downloads/
https://alpinelinux.org/releases/

yet it gives,
```
2021-09-07T03:40:48.846Z	WARN	This OS version is not on the EOL list: alpine 3.14
2021-09-07T03:40:48.846Z	INFO	Detecting Alpine vulnerabilities...
2021-09-07T03:40:48.848Z	WARN	This OS version is no longer supported by the distribution: alpine 3.14.2
2021-09-07T03:40:48.848Z	WARN	The vulnerability detection may be insufficient because security updates are not provided
Total: 0 (MEDIUM: 0, HIGH: 0, CRITICAL: 0)
```

## Output of run with `-debug`:

```
(paste your output here)
```

## Output of `trivy -v`:

```
(paste your output here)
```

**Comment by czunker** [comment]
I cannot reproduce the problem:
```
trivy -debug image alpine:3.14.2                                                                           
2021-09-30T14:10:00.258+0200	DEBUG	Severities: UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL
2021-09-30T14:10:00.264+0200	DEBUG	cache dir:  /home/czunker/.cache/trivy
2021-09-30T14:10:00.264+0200	DEBUG	DB update was skipped because DB was downloaded during the last hour
2021-09-30T14:10:00.264+0200	DEBUG	DB Schema: 1, Type: 1, UpdatedAt: 2021-09-30 06:05:17.992044192 +0000 UTC, NextUpdate: 2021-09-30 12:05:17.992043692 +0000 UTC, DownloadedAt: 2021-09-30 11:14:42.878341288 +0000 UTC
2021-09-30T14:10:00.264+0200	DEBUG	Vulnerability type:  [os library]
2021-09-30T14:10:01.874+0200	DEBUG	Image ID: sha256:14119a10abf4669e8cdbdff324a9f9605d99697215a0d21c360fe8dfa8471bab
2021-09-30T14:10:01.874+0200	DEBUG	Diff IDs: [sha256:e2eb06d8af8218cfec8210147357a68b7e13f7c485b991c288c2d01dc228bb68]
2021-09-30T14:10:01.876+0200	INFO	Detected OS: alpine
2021-09-30T14:10:01.876+0200	INFO	Detecting Alpine vulnerabilities...
2021-09-30T14:10:01.876+0200	DEBUG	alpine: os version: 3.14
2021-09-30T14:10:01.876+0200	DEBUG	alpine: the number of packages: 14
2021-09-30T14:10:01.878+0200	INFO	Number of language-specific files: 0

alpine:3.14.2 (alpine 3.14.2)
=============================
Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)
```

@AmaliMatharaarachchi Could you please provide more information (trivy version, DB version)? Or give it a try with the latest trivy and database?

**Comment by knqyf263** [comment]
We've released v0.20.0. Please try it out.

```
$ trivy image alpine:3.14
2021-10-06T13:47:02.648+0300    INFO    Detected OS: alpine
2021-10-06T13:47:02.648+0300    INFO    Detecting Alpine vulnerabilities...
2021-10-06T13:47:02.652+0300    INFO    Number of language-specific files: 0

alpine:3.14 (alpine 3.14.2)
===========================
Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)
```

**Comment by knqyf263** [comment]
Please feel free to reopen if you still face the same issue with v0.20.0.
